src/2018/transitioning/errors/question-mark.md

```diff
@@ -1,6 +1,6 @@
 # `?` in `fn main()` and `#[test]`s
 
-Typically in Rust, error handling revolves around returning `Result` and using
+Rust's error handling revolves around returning `Result` and using
```

Comment:
Is this not overstating things? `Option<T>` can work well in some cases as well.
Perhaps the simplification is useful for the purposes of teaching tho.

Reply:
Option isn't about error handling, IMHO.